scripts/get_coldcard_address.js

```diff
@@ -0,0 +1,8 @@
+const { get_address_from_coldcard } = require('../hsm')
+
+require('dotenv').config({
+    override: true,
+})
+;(async () => {
+    console.log(await get_address_from_coldcard())
```

Comment:
@topether21 need to pass in the derivation path here

Reply:
![image](https://github.com/deezy-inc/sat-hunter/assets/126987350/c12abb44-f726-4d38-be6c-6766069e7b01)
![image](https://github.com/deezy-inc/sat-hunter/assets/126987350/2e05d273-2462-4bc3-84ec-ee878b570465)

Reply:
nice!